libs/application/templates/aosh/seminars/src/shared/types.ts

```
@@ -0,0 +1,21 @@
+import { z } from 'zod'
+import {
+  ParticipantSchema,
+  PaymentArrangementSchema,
+  SeminarAnswersSchema,
+} from '../lib/dataSchema'
+import { MessageDescriptor } from 'react-intl'
+
+export type SeminarAnswersSchema = z.TypeOf<typeof SeminarAnswersSchema>
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SeminarAnswersSchema'. Consider to delete it or rename it.

'SeminarAnswersSchema' is defined here:


(lint/suspicious/noRedeclare)